src/stdlib/distributions.jl

```diff
@@ -49,3 +49,52 @@ end
 function Distributions.logpdf(d::VecBinomialLogit{<:Real}, ks::Vector{<:Integer})
     return sum(logpdf_binomial_logit.(d.n, d.logitp, ks))
 end
+
+
+struct OrderedLogistic{T1, T2} <: DiscreteUnivariateDistribution
```

Comment:
Since we're storing the `cutpoints` regardless their type, might want to consider parametrising the entire type of `cutpoints`. i.e.
```julia
struct OrderedLogistic{T1, T2<:AbstractVector} <: DiscreteUnivariateDistribution
   η::T1
   cutpoints::T2

   function OrderedLogistic(η, cutpoints)
        if !issorted(cutpoints)
            error("cutpoints are not sorted")
        end
        return new{typeof(η), typeof(cutpoints)}(η, cutpoints)
   end

end
```

Reply:
This gives `invalid redefinition of constant OrderedLogistic`. I tried to make some minor corrections, still getting error messages.

Reply:
Hmmm weird, seems to work fine for me. Is this a `Revise.jl` issue? If so, try restarting your session.